CHANGELOG.rst

```diff
@@ -365,3 +365,18 @@ New Modules
 - fact_diff - Find the difference between currently set facts
 - update_fact - Update currently set facts
 - validate - Validate data with provided criteria
+
+v4.0.0
+======
+
+Major Changes
+-------------
+
+- This release removes deprecated is_private() function in favour of netaddr is_global(). Please refer to the **Removed Features** section for details.
+
+Removed Features (previously deprecated)
+----------------------------------------
+
+- Remove deprecated is_private() function which is replaced with netaddr is_global() function.
+- Remove the IPAddress.is_private method.
+- Set netaddr minimum version required to 0.10.1
```

Comment:
Can you move this line to major changes?

Overall, from the release notes it isn't clear what user facing  features are being impacted with this major release. In fact, even the affected filter plugin isn't mentioned here. This only talks about code change, which is abstracted to the user anyway.

Reply:
I don't think you need a "Removed Features" section here. Rather, an explanation in "Major Changes" about _why_ we had to merge https://github.com/ansible-collections/ansible.utils/pull/338.